assets/js/components/notifications/ZeroDataStateNotifications/index.js

```diff
@@ -37,8 +38,17 @@ const { useSelect, useInViewSelect } = Data;
 export default function ZeroDataStateNotifications() {
 	const viewOnly = useViewOnly();
 	const isAnalyticsConnected = useSelect( ( select ) =>
-		select( CORE_MODULES ).isModuleConnected( 'analytics' )
+		select( CORE_MODULES ).isModuleConnected(
+			isGA4DashboardView ? 'analytics-4' : 'analytics'
+		)
 	);
+	const isGA4DashboardView = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).isGA4DashboardView()
+	);
```

Comment:
It looks like `isGA4DashboardView` is referenced in the `useSelect` function before it's defined. We should probably define it first, just to make reading this easier—though also to avoid any bugs/errors.

Reply:
That's a good catch 👍 It wasn't caught due to `variables: false`:
https://github.com/google/site-kit-wp/blob/f2debd81a7dbcfe9a3dc62c954d7ccd18e9c0582/.eslintrc.json#L54-L61

Reply:
Huh! I checked into disabling that `variables: false` bit and it seems a bit involved, so we'll leave it as-is for now 👍🏻